runtime/js.rs

```diff
@@ -1,14 +1,33 @@
 // Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
 use deno_core::Snapshot;
 use log::debug;
+use once_cell::sync::Lazy;
 
-pub static CLI_SNAPSHOT: &[u8] =
-  include_bytes!(concat!(env!("OUT_DIR"), "/CLI_SNAPSHOT.bin"));
+pub static CLI_SNAPSHOT: Lazy<Box<[u8]>> = Lazy::new(
+  #[cold]
+  #[inline(never)]
+  || {
+    static COMPRESSED_CLI_SNAPSHOT: &[u8] =
+      include_bytes!(concat!(env!("OUT_DIR"), "/CLI_SNAPSHOT.bin"));
+
```

Comment:
I'm confused now, weren't we supposed to not compress CLI snapshot to avoid decompression penalty? Or is it so insignificant?

Reply:
compression is practically free because of how fast lz4 is
you can see the difference between lz4 and uncompressed in first benchmark